packages/Checkbox/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@paprika/checkbox",
+  "version": "0.1.0",
+  "description": "Checkbox component",
+  "author": "@paprika",
+  "license": "MIT",
+  "main": "lib/index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/acl-services/paprika/tree/master/packages/checkbox"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@babel/runtime-corejs2": "^7.3.1",
+    "@paprika/helpers": "^0.2.0",
+    "@paprika/icon": "^0.2.0",
+    "@paprika/stylers": "^0.1.5",
+    "@paprika/tokens": "^0.1.4",
+    "prop-types": "^15.7.2",
+    "uuid": "^3.3.2"
+  },
+  "devDependencies": {
```

Comment:
As long as these are in the main package.json, I don't think they're required here.